tools/bash_completion

```diff
@@ -0,0 +1,43 @@
+__gohan()
+{
+
+
+    declare -A possibleWord=( ["1"]="gohan"
+    ["2"]="--debug --help --version -d -h -v"
+    ["3"]="client validate v init-db idb convert conv server srv test_extensions test_ex migrate mig template run test test openapi markdown dot glace-server gsrv help h")
```

Comment:
I couldn't get both `bash_completion` and `gohan_client_bash_completion.sh` to work at the same time. Sourcing one made the other have no effect, and vice-versa.